### PR TITLE
(maint) Fix some typos in the readme

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,16 +25,27 @@ This module installs and configures [Kubernetes](https://kubernetes.io/). Kubern
...
+
 ## Setup
 
 ### Setup Requirements
 
-This module includes a configuration tool called `kubetool` to auto generate
-all the security parameters, the bootstrap token, and other configurations for
-your Kubernetes cluster into a Hiera file. The tool is available as a Docker
+This module includes a configuration tool called
+[kubetool](https://github.com/abema/kubetool) to auto generate all the security
```
Comment (ccaum) [inline README.md]: Not that kubetool :)
Our kubetool is specific to the module and in the tooling/kube_tool directory.

**Comment by lucywyman** [comment]
Oh my goodness, I completely forgot about this! Apologies! @ccaum fixed :)

**Comment by ccaum** [comment]
Oh wow I forgot about this too. Thanks @lucywyman!  @davejrt or @scotty-c, LGTM. Can you merge?

**Comment by scotty-c** [comment]
LGTM

**Pull request merged**